### PR TITLE
To support Java and .NET universal instrument

### PR DESCRIPTION
**Pull request by nine5two7**
### What does this PR do?
To support Java and .NET universal instrument.
Point to correct extension version and dd-trace version.
Set the aws_lambda_exec_wrapper env variable.
The newest published versions of  dd-extension, dd-trace-dotnet, and dd-trace-java can met the requirement.

### Motivation
https://datadoghq.atlassian.net/browse/SLS-2202
To provide to our customers with more convenient way to configure DD serverless for Java, and .NET runtime.

### Testing Guidelines
Unit Test and Test with our sample Lambda functions
Manual tests have been done vie the below two lambda fucntions
[Java](https://sa-east-1.console.aws.amazon.com/lambda/home?region=sa-east-1#/functions/lambda-java-universal-9527-0727-dev-helloEnd?tab=configure)
[.NET](https://sa-east-1.console.aws.amazon.com/lambda/home?region=sa-east-1#/functions/test-dotnet6-9527-0727-dev-handlerAPIGatewayOnly?tab=configure)

### Types of changes

- [ ] Bug fix
- [x] New feature
- [ ] Breaking change
- [ ] Misc (docs, refactoring, dependency upgrade, etc.)

### Check all that apply

- [x] This PR's description is comprehensive
- [ ] This PR contains breaking changes that are documented in the description
- [ ] This PR introduces new APIs or parameters that are documented and unlikely to change in the foreseeable future
- [x] This PR impacts documentation, and it has been updated (or a ticket has been logged)
- [ ] This PR's changes are covered by the automated tests
- [ ] This PR collects user input/sensitive content into Datadog

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -390,7 +390,7 @@ module.exports = class ServerlessPlugin {
 
   /**
    * Uploads git metadata for the current directory to Datadog and goes through
-   * each function defined in serverless and attaches the git.commit.sha to DD_TAGS.
+   * each function defined in serverless and attaches git.commit.sha and git.repository_url to DD_TAGS.
```
Comment (nine5two7) [inline src/index.ts:393]: Code diff as shown in this file is from git rebase after creating the PR.

File: src/source-code-integration.ts [hunk 2]
```diff
@@ -48,7 +48,7 @@ export class SourceCodeIntegration {
         throw new Error("Error uploading commit information.");
       }
 
-      return payload.hash;
+      return { hash: payload.hash, remote: payload.remote };
```
Comment (nine5two7) [inline src/source-code-integration.ts:51]: Code diff as shown in this file is from git rebase after creating the PR.

**Review by maxday — COMMENTED** [review]
this looks good to me!
2 small comments :

- I think extension version will be v24 (not v25)
- Can you merge main into this branch so unrelated diff won't show up?

**Comment by nine5two7** [comment]
@maxday, here is the new revision.

**Review by jcstorms1 — APPROVED** [review]
LGTM 👍

**Pull request merged**